src/ui/molecules/stripe-elements.svelte

```diff
@@ -194,120 +176,93 @@
       return;
     }
 
-    const { countryCode, postalCode } =
-      getCountryAndPostalCodeFromConfirmationToken(confirmationToken);
+    const billingAddress =
+      confirmationToken.payment_method_preview?.billing_details?.address;
 
     if (
-      countryCode === lastTaxCustomerDetails?.countryCode &&
-      postalCode === lastTaxCustomerDetails?.postalCode
+      !billingAddress ||
+      JSON.stringify(billingAddress) === JSON.stringify(lastTaxCustomerDetails)
```

Comment:
We are comparing a type returned by Stripe with a custom type. If any of these types change, it will break this behavior.

I’m a bit concerned about introducing behavioral changes in this PR, since we’re still missing integration tests for the tax calculation part and could unintentionally introduce breaking changes.

Reply:
I'll check the specific fields just to be sure